server/api.ts

```diff
@@ -17,5 +17,11 @@ export interface GetPostResponse {
 // Comment APIs
 
 // Like APIs
+export type CreateLikeRequest = Like
+export interface CreateLikeResponse {}
+export type GetLikesRequest = String;
```

Comment:
Let's wrap this in an object, similar to other request types, so we can add more fields to it later if needed:
```
export interface GetLikesRequests {
  postId: string;
}
```

Reply:
that's much more descriptive.
Thanks